translation/translator.go

```diff
@@ -81,37 +82,38 @@ func (t *Translator) Get(key string, options ...translationcontract.Option) stri
 		fallback = *options[0].Fallback
 	}
 
-	// Parse the key into group and key parts.
-	group, keyPart := parseKey(key)
-
 	// For JSON translations, there is only one file per locale, so we will
 	// simply load the file and return the line if it exists.
 	// If the file doesn't exist, we will return fallback if it is enabled.
 	// Otherwise, we will return the key as the line.
-	if err := t.load(locale, group); err != nil && err != ErrFileNotExist {
+	if err := t.load(locale, "*"); err != nil && err != ErrFileNotExist {
```

Comment:
The `facades.Lang` is not a single instance, so the locale file will be loaded every time, we can set the `loaded` map to a global variable, and the performance will be improved.

Reply:
?

Reply:
We are not support "/" explicitly just like laravel.